cpp/src/glm/ridge.cuh

```diff
@@ -37,7 +37,7 @@ namespace GLM {
 using namespace MLCommon;
 
 template <typename math_t>
-void ridgeSolve(const cumlHandle_impl &handle, math_t *S, math_t *V, math_t *U,
+void ridgeSolve(const handle_impl &handle, math_t *S, math_t *V, math_t *U,
```

Comment:
I didn't realize there were so many places that relied specifically on the impl :-/ Not sure why this was necessary.

Reply:
It's unfortunate. Wish these were just passing the handle along. I suppose I could refactor that but that would be much longer.